jwt/api_jwt.py

```diff
@@ -55,7 +66,12 @@ def encode(self, payload, key, algorithm='HS256', headers=None,
             json_payload, key, algorithm, headers, json_encoder
         )
 
-    def decode(self, jwt, key='', verify=True, algorithms=None, options=None,
```

Comment:
change jwt -> token, changes public interface and broken my code

Reply:
Sorry, I didn't see it as a major change as the argument is positional, turns out I was wrong and there are users with the same problem. It has been reverted [here](https://github.com/jpadilla/pyjwt/issues/351)